packages/dartcv/test/imgcodecs_test.dart

```diff
@@ -45,6 +55,30 @@ void main() async {
     expect(dst.isEmpty, false);
     expect((dst.rows, dst.cols, dst.channels), (cvImage.rows, cvImage.cols, cvImage.channels));
   });
+
+  test("cv2.imencodeVec, cv2.imdecodeVec", () async {
+    final cvImage = cv.imread("test/images/circles.jpg", flags: cv.IMREAD_COLOR);
+    expect((cvImage.width, cvImage.height), (512, 512));
+    final (success, buf) = cv.imencodeVec(".png", cvImage);
+    expect(success, true);
+    expect(buf.length, greaterThan(0));
+    await File("test/images_out/test_imencode.png").writeAsBytes(buf.data);
+    final params = [cv.IMWRITE_PNG_COMPRESSION, 9].i32;
+    final (success1, buf1) = cv.imencodeVec(".png", cvImage, params: params);
+    expect(success1, true);
+    expect(buf1.length, greaterThan(0));
+
+    final cvimgDecode = cv.imdecodeVec(buf, cv.IMREAD_COLOR);
+    expect(cvimgDecode.height, equals(cvImage.height));
+    expect(cvimgDecode.width, equals(cvImage.width));
+    expect(cvimgDecode.channels, equals(cvImage.channels));
+
+    final dst = cv.Mat.empty();
+    cv.imdecodeVec(buf, cv.IMREAD_COLOR, dst: dst);
+    expect(dst.isEmpty, false);
+    expect((dst.rows, dst.cols, dst.channels), (cvImage.rows, cvImage.cols, cvImage.channels));
+  });
```

Comment:
Should the buffer here be disposed? Or is that handled by the test framework / not matter?

Reply:
Better to dispose it manually.

But here only several `VecUChar` objects will be created and they are not very large, since they are attacted to dart objects via `NativeFinalizer`, letting dart VM dispose them automatically is okay, but if many such objects will be created in your project, it's necessary to dispose them manually.